connect/src/state.rs

```diff
@@ -141,35 +148,24 @@ impl ConnectState {
                     connect_disabled: false,
                     supports_rename: false,
                     supports_external_episodes: false,
-                    supports_set_backend_metadata: false, // TODO: impl
+                    supports_set_backend_metadata: false,
                     supports_hifi: MessageField::none(),
 
                     command_acks: true,
                     ..Default::default()
                 }),
                 ..Default::default()
             },
+            prev_tracks: VecDeque::with_capacity(SPOTIFY_MAX_PREV_TRACKS_SIZE),
```

Comment:
Just dropping what you may already know: this will pre-allocate enough memory for the `VecDeque` to hold that many items, but it won't be a bounded queue.

Reply:
Yes, I mean with a fixed size. I don't think you can do that without an array, some crate or an own implementation.